frontend-student/src/components/coursePage/NotesSection.jsx

```
@@ -47,27 +50,19 @@ export default function NotesSection({ videoId }) {
 
   const handleSaveNote = async () => {
     if (!videoId || !note.trim()) return;
-    
+
     setIsLoading(true);
     try {
       let noteData = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'noteData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)